Add sudo to the shell commands

obadz opened the pull request:
..so that they can be readily pasted

Commit:
Add sudo to the shell commands

..so that they can be readily pasted

txdv commented:
Depends on the system. Easier to use doesn't mean safer to use.

akoeplinger commented:
I'm a bit torn on this one. Debian doesn't have sudo by default (so users there can't copy/paste either), which means the only real argument in favor of sudo I have is that Ubuntu is the go-to distribution for users that are new to Linux, so they might not immediately know what "run this in a root shell" means.

Debian users on the other hand will more likely know that they need to remove `sudo` from commands and switch to a root shell instead.

I guess I'm slightly :+1: for taking this change, what do others think?

obadz commented:
I think sudo has become the standard way to encode into publicly posted commands that they should be run as root. Ubuntu & Mint ship it by default and I think Debian has it in the standard system utilities task? (isn't that installed unless specifically unchecked by user during install?)

akoeplinger commented on docs/getting-started/install/linux/index.md:
```diff
@@ -14,16 +14,16 @@ Regardless of your distribution, you will need the Mono Project GPG signing key,
 -------------------------------
 
-Add the Mono Project GPG signing key **in a root shell** with:
+Add the Mono Project GPG signing key with (if you don't use sudo, be sure to switch to root):
```
any reason for `with`? I think it reads better without.

akoeplinger commented:
Please also change the other occurences of > to | sudo tee.

Commit:
Update index.md

Remove the with
Change the other > to sudo tee
Add comment about switching to root everywhere

akoeplinger commented:
Thanks! If you have time, you could make the same changes to the MonoDevelop guide, to keep everything consistent: https://github.com/mono/md-website/blob/gh-pages/download/linux.md
